src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -400,7 +399,8 @@ export default class DashboardTable extends React.Component {
                         onSort={this.sortTable.bind(this)} />
                 </div>
                 <div className="text-center">
-                    {this.state.message}
+                    {this.state.noResults && <NoResultsMessage />}
+                    {this.props.isLoading && <LoadingMessage />}
                 </div>
```

Comment:
👍 I like this approach better. I guess formerly we were doing a bunch of CSS magic based on the class value in `line 393`?

Reply:
we were adding a class to the table header based on loading or not loading or results. I originally deleted that, however, that was taking care of the opacity in the table header which is needed to make the design.  I added it back in with `classnames` npm package and `tableClasses` method